NEWS

```diff
@@ -1499,3 +1499,22 @@ overdispersion parameters
 
 version 2.1.7 (2020-09-01)
 - Bugfix in format method for ResultVG class.
```

Comment:
This should be

```
version 2.1.8 (2020-09-23)
- list of changes
```

Reply:
2.1.6 is the latest on Bioc, I was going by that

Reply:
(sorry to be painful about this - trying to be consistent with BioC rules)

We should bump z for *every* commit. Technically, we are now in version 2.1.? :-)

Reply:
... I've lost the count about how many commits!

Reply:
In that case it's maybe better to merge and then I'll rebase - shouldn't be hard and is really much easier to track things that way

For future PRs it's probably better to squash and merge

Reply:
OK. 

And agreed about future PR.